pandas/tests/test_multilevel.py

```diff
@@ -2381,6 +2381,29 @@ def test_iloc_mi(self):
         tm.assert_frame_equal(result, expected)
 
 
+class TestSparse(object):
```

Comment:
I could. But with pytest style promoting flat, module-level functions only, the module namespace will get polluted fast and often, so the tests modules will likely benefit from becoming more fine grained.